repo/packages/C/calico/0/marathon.json.mustache

```diff
@@ -13,8 +13,8 @@
     "CALICO_CALICOCTL_URL": "{{calico.calicoctl-url}}",
     "CALICO_NODE_IMG": "{{calico.calico-node-img}}",
     "CALICO_LIBNETWORK_IMG": "{{calico.calico-libnetwork-img}}",
-    "CALICO_ALLOW_RESTART_AGENT": "{{calico.allow-restart-agent}}",
-    "CALICO_ALLOW_RESTART_DOCKER": "{{calico.allow-restart-docker}}",
+    "CALICO_ALLOW_AGENT_UPDATE": "{{calico.allow-restart-agent}}",
```

Comment:
Name changes are fine - but when renaming stuff - check whether other things need updating such as comments, log strings, doc strings ... and in this case other variable names ... allow-restart-agent   ->  allow-update-agent